assets/css/base/_base.scss

```diff
@@ -85,3 +86,12 @@ button:-moz-focusring {
   color: transparent;
   text-shadow: 0 0 0 $grey-dark;
 }
+
+// News
```

Comment:
Please remove the commented css and code. This comment applies to the entire pull request.